ArchiSteamFarm/Program.cs

```diff
@@ -66,12 +68,20 @@ internal static class Program {
 				return null;
 			}
 
+			string result;
+
 			if (GlobalConfig.Headless || !Runtime.IsUserInteractive) {
+				var userInputKey = new KeyValuePair<string, ASF.EUserInputType>(botName, userInputType);
+				if (UserInputs.ContainsKey(userInputKey)) {
```

Comment:
On the second look it even must be done like I said, since we don't control **at all** to which bot we actually assign given input, and it must be possible to control. My solution would solve that issue.